ydb/core/tx/schemeshard/schemeshard__operation_modify_acl.cpp

```diff
@@ -51,6 +51,26 @@ class TModifyACL: public TSubOperationBase {
             return result;
         }
 
+        if (acl) {
```

Comment:
these checks don't make any sense:
1. the idea behind YDB ACLs is that you can grant permissions to any SID (whether it exists or not). This is intentional, as a user can be created after their permissions are set.
2. You can remove a user after this check, making it redundant.

Reply:
1. https://st.yandex-team.ru/KIKIMR-22389
2. https://st.yandex-team.ru/KIKIMR-22266